Foundation/Unit.swift

```diff
@@ -332,14 +986,48 @@ public class UnitFuelEfficiency : Dimension {
      Base unit - litersPer100Kilometers
      */
     
-    public class var litersPer100Kilometers: UnitFuelEfficiency { get { NSUnimplemented() } }
+    private struct Symbol {
+        static let litersPer100Kilometers   = "L/100km"
+        static let milesPerImperialGallon   = "mpg"
+        static let milesPerGallon           = "mpg"
+    }
     
-    public class var milesPerImperialGallon: UnitFuelEfficiency { get { NSUnimplemented() } }
+    private struct Coefficient {
+        static let litersPer100Kilometers   = 1.0
+        static let milesPerImperialGallon   = 282.481
+        static let milesPerGallon           = 235.215
+    }
     
-    public class var milesPerGallon: UnitFuelEfficiency { get { NSUnimplemented() } }
+    private init(symbol: String, coefficient: Double) {
+        super.init(symbol: symbol, converter: UnitConverterLinear(coefficient: coefficient))
+    }
+    
+    
+    public class var litersPer100Kilometers: UnitFuelEfficiency {
+        get {
+            return UnitFuelEfficiency(symbol: Symbol.litersPer100Kilometers, coefficient: Coefficient.litersPer100Kilometers)
+        }
+    }
+    
+    public class var milesPerImperialGallon: UnitFuelEfficiency {
+        get {
+            return UnitFuelEfficiency(symbol: Symbol.milesPerImperialGallon, coefficient: Coefficient.milesPerImperialGallon)
```

Comment:
@parkera shouldn't be like this?

`- (double)valueFromBaseUnitValue:(double)baseUnitValue {
    return baseUnitValue * _reciprocalValue;
}`